content/posts/2021/reana-client-python-api.md

```diff
@@ -0,0 +1,135 @@
+---
+title: How to run workflows using reana-client Python API"
+date: 2021-02-22T15:00:00+01:00
+tags:
+  - reana-client
+---
+
+Here you can find a step by step guide on how to submit workflows to
+REANA using the `reana-client` Python API. This is a simple example,
+for illustrative purposes, using the serial workflow engine and a
+simple `ROOT6/RooFit` analysis example.
+<!--more-->
+
+Install `reana-client`:
+```
+$ pip install reana-client==0.7.0
+```
+
+And now, inside a Python interpreter, or a Python file:
+```python
+
+# Let us use the REANA example - ROOT6/ RooFit analysis
```

Comment:
Can you please break these comments into regular blog post text, so that there would be many blocks of leading text and Python code snippets?  (A more usual blog style.)

Reply:
... and the blog post could end with displaying the final graph.

Reply:
Sure, the reason I initially kept the code in a single block is that now it's very easy to copy the code to a Python script and simply execute it

Reply:
It's a valid point, what about adding a link to a gist at the bottom of the blog post?

> - Access the full Python script here.

or similar.

Reply:
We could in this case create a Jupyter notebook.  It would have both nice presentation and be fully runnable.

(The advantage of a blog post is that it allows to make detours, to say "what if", to present alternatives... Basically just to teach, not to be blindly copied...)